ui/src/app/job-list/table/table.component.ts

```diff
@@ -200,4 +200,26 @@ export class JobsTableComponent implements OnInit {
       this.jobs.forEach(j => this.selection.select(j));
     }
   }
+
+  updateCheckBoxSelection(clickedJob: QueryJobsResult, event: MouseEvent): void {
+    // if the user has shift-clicked on a job, find the last selected job and
+    // select all jobs between the two
+    if (event.shiftKey && this.selection.selected.length) {
```

Comment:
nit: `.length` sort of redundant, `[]` is falsey in JS

alternatively: `this.selection.hasValue()`

Reply:
agreed – done